webapp/src/components/PrivacyPolicyContent.js

```diff
@@ -424,6 +344,56 @@ export default function PrivacyPolicyContent({ mobile }) {
 const TableOfContentsNav = ({ id, mobile }) => {
   mobile = Boolean(mobile);
   const [expanded, setExpanded] = React.useState(!mobile || false);
+  const navLinks = [
```

Comment:
Pull this to the module level so it isn't recalculated on each render. I don't know if JS is smart enough to cache it.